### PR TITLE
Yield all the other markers from ember-leaflet

### PR DESCRIPTION
**Pull request by bzf**
Copied them over from the [`ember-leaflet` repo][1].

[1]: https://github.com/miguelcobain/ember-leaflet/blob/32d13608d88f576da934f0e2f4217353fafeeb97/addon/templates/leaflet-map.hbs

### CODE REVIEW

**Comment by miguelcobain** [comment]
Why do you need this? A marker cluster should only have markers, not other layers.

**Comment by bzf** [comment]
The reason I added these was so that I could use the `circle-marker` when zooming in on the map instead of the regular `marker` one.

## From this:
Screenshot 2020-09-18 at 07 49 49


```hbs
  <layers.marker-cluster as |cluster|>
    {{#each @data as |row|}}
      <cluster.marker @location={{array row.longitude row.latitude}} as |marker| >
        <marker.popup>
          <span>
            <strong>{{row.name}}:</strong>
            <span>{{row.value}}</span>
          </span>
        </marker.popup>
      </cluster.marker>
    {{/each}}
  </layers.marker-cluster>
```

## To this:
Screenshot 2020-09-18 at 07 40 18

```hbs
  <layers.marker-cluster as |cluster|>
    {{#each @data as |row|}}
      <cluster.circle-marker @location={{array row.longitude row.latitude}} as |marker| >
        <marker.popup>
          <span>
            <strong>{{row.name}}: </strong>
            <span>{{row.value}}</span>
          </span>
        </marker.popup>
      </cluster.circle-marker>
    {{/each}}
  </layers.marker-cluster>

```

Maybe that can be achieved some other way?

**Comment by bzf** [comment]
@miguelcobain Sorry to ping, but have you had time to think over my response? 😊

**Comment by miguelcobain** [comment]
@bzf your suggestion makes sense. Do you mind to use the latest ember-leaflet beta version and ember-leaflet-marker-cluster latest version?

**Comment by bzf** [comment]
@miguelcobain Not at all, I'll push an update during the day 😊

**Comment by bzf** [comment]
Moved the markers over to the JS file and tried it out with the `v5.0.0-beta3` release of `ember-leaflet` and seemed to work in our application. :)

File: addon/components/marker-cluster-layer.js [hunk 1]
```diff
@@ -52,6 +52,18 @@ export default class MarkerClusterLayer extends BaseLayer {
   componentsToYield = [
     ...this.componentsToYield,
     { name: 'marker-layer', as: 'marker' },
+    { name: 'tile-layer', as: 'tile' },
```
Comment (miguelcobain) [inline addon/components/marker-cluster-layer.js:55]: @bzf what layers can be added to a marker cluster?

I'm pretty sure you can't add tile-layers to marker clusters, right?

Comment (bzf) [reply addon/components/marker-cluster-layer.js:55]: TBH, I don't know. I think I just moved over everything from https://github.com/miguelcobain/ember-leaflet/blob/32d13608d88f576da934f0e2f4217353fafeeb97/addon/templates/leaflet-map.hbs, which seems pretty wrong.

I'm only using the `circle-marker` one in my scenario - maybe it's smarter to only add the one I know for sure works?

**Comment by miguelcobain** [comment]
This was released on `1.0.1` version. Thank you!

**Pull request merged**